packratt/tests/conftest.py

```diff
@@ -38,9 +39,14 @@ def test_cache(tmp_path_factory):
 def registry(tmp_path_factory):
 
     user_conf_path = tmp_path_factory.mktemp("conf")
-    USER_REGISTRY = user_conf_path / "registry.yaml"
-    CONTENT = yaml.dump(content)
-    USER_REGISTRY.write_text(CONTENT)
-    assert USER_REGISTRY.read_text() == CONTENT
+    this_registry.USER_REGISTRY = user_conf_path / "registry.yaml"
```

Comment:
I think this will clobber the user registry on any machine on which the tests are run.

Does unittest.mock.patch not work?

Reply:
Did try `unittest.mock.patch`. Let me have a look at it